### DISCUSSION TITLE
Column Type becomes un editable

### DISCUSSION THREAD

**Discussion by ekansh005** [Budibase/budibase#3897] [Help]
Hi,
When you edit a column in Budibase DB and select 'Choose an option' in the type dropdown, now it is not possible to select the type again unless you refresh the whole page.
BR, Ekansh

**Comment by ekansh005** [comment]
Hi,
I understand that holiday season has just ended but do we have any update on this issue?

BR, Ekansh

**Comment by mjashanks** [comment]
Hi Ekansh. Thanks for reporting this. An easy workaround is to close the modal and edit again. I'll log an issue for this anyway.
